dev-client/jest/unit/setup.ts

```diff
@@ -15,4 +15,11 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-// fill me in if you need special config for unit testing
+let mmkvMock = require('react-native-mmkv-storage/jest/dist/jest/memoryStore.js');
+mmkvMock.mock(); // Mock the storage
+
+beforeEach(() => {
```

Comment:
I don't know if I agree with making this a globally-configured setting in our unit tests; is it possible to just set this up for the kv storage and feature flag test files? The logic seems so minimal and we don't know what the cost is of clearing the mmkv mock for every test case.

Reply:
i looked into it a bit, it looks like the cost of clearing the mock is more or less setting a handful of pointers to null and leaving some objects to be GC'd

it is possible to do it in individual test files, but it can be awkward because you need to manually make sure the imports resolve in the right order since the mock expects to be required in a setup file like this before any of the library's use sites are resolved

if it turns out to be a problem it should be straightforward to manually inline it though